Scripts/application.py

```diff
@@ -0,0 +1,112 @@
+import sqlite3
+import json
+from typing import Dict
+from flask import Flask, request
+app = Flask(__name__)
+
+path_to_db = '../db/allYourSantaAreBelongToUs.db'
+
+@app.route('/')
+def hello_world():
+    return 'Hello World!'
+    
+@app.route('/api/v1/person', methods = ["GET","POST"])
+def return_all_people():
+    
+    if request.method == "POST":
+        posted_email = request.get_json()['email']
+        with sqlite3.connect(path_to_db) as connection:
+            cursor = connection.cursor()
+            cursor.execute('INSERT into people (email) VALUES (?);', (posted_email,))            
+    
+    # returns a json of all people
+    with sqlite3.connect(path_to_db) as connection:
+        cursor = connection.cursor()
+        #get all of the people
+        cursor.execute('SELECT person_ID, email, givingTo_ID FROM people')
+        raw_data: List = cursor.fetchall()
+        data: Dict = {}
+        for person in raw_data:
+            data[person[0]] = {"email": person[1], "assignment ID": person[2]}
+        json_data = json.dumps(data)
+    return json_data

```

Comment:
you don't need to return the entire json_data for POST requests. It's better to just return the single person that was created so the caller can use the id if needed.

Reply:
Yes.